tox.ini

```diff
@@ -42,3 +43,11 @@ deps =
     django-stubs==4.2.7
 commands =
     mypy solo
+
+[testenv:lint]
+skip_install = true
+deps =
+    ruff==0.4.6
```

Comment:
Any particular reason not to use `0.5.0` here?

Reply:
`0.4.6` was latest when I created the PR. Updated.